go.sum

```diff
@@ -153,6 +153,8 @@ github.com/cockroachdb/datadriven v0.0.0-20190809214429-80d97fb3cbaa/go.mod h1:z
 github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd/go.mod h1:sE/e/2PUdi/liOCUjSTXgM1o87ZssimdTWN964YiIeI=
 github.com/coinbase/rosetta-sdk-go v0.7.0 h1:lmTO/JEpCvZgpbkOITL95rA80CPKb5CtMzLaqF2mCNg=
 github.com/coinbase/rosetta-sdk-go v0.7.0/go.mod h1:7nD3oBPIiHqhRprqvMgPoGxe/nyq3yftRmpsy29coWE=
+github.com/confio/ics23/go v0.7.0 h1:00d2kukk7sPoHWL4zZBZwzxnpA2pec1NPdwbSokJ5w8=
```

Comment:
wait, we still want to force this to 0.8.0, no?

Reply:
https://github.com/cosmos/ibc-go/blob/8db37635365d87f9dffc2fdc1b55ffb83f461ffa/go.mod#L129
`ibc-go` still has this replace.

So I think we still need the replacement line. @the-frey thanks for your catch.

Reply:
no sweat!